### ISSUE TITLE
University of Puerto Rico Mayaguez, University of Puerto Rico Mayaguez Invertebrate Collection: Update Request

### ISSUE THREAD

**Issue by iDigBioBot** [iDigBio/idb-us-collections#168]
```json 
{
	"institution": "University of Puerto Rico Mayaguez",
	"collection": "University of Puerto Rico Mayaguez Invertebrate Collection",
	"recordsets": "",
	"institution_code": "UPRM",
	"collection_code": "INVCOL",
	"collection_uuid": "urn:uuid:f2f50ab7-ac4a-430e-9c15-7e389a4bffdf",
	"collection_lsid": "NA",
	"collection_url": "http://alexvandam.wix.com/uprminvcol",
	"collection_catalog_url": "https://www.idigbio.org/portal/recordsets/2b89de41-42bd-46c6-ab61-d386f855f7fb",
	"contact": "Alex Van Dam",
	"contact_role": "Director",
	"contact_email": "alex.vandam@upr.edu",
	"taxonomic_coverage": "Arthropoda and Mollusca",
	"geographic_range": "Mostly the Caribbean Islands\n",
	"collection_size": "150,000",
	"mailing_address": "Departamento de Biología, Call Box 9000",
	"mailing_city": "Mayaguez",
	"mailing_state": "Puerto Rico",
	"mailing_zip": "00681-9000",
	"physical_address": "Departamento de Biologia, Carr. 108 Barrio Miradero Km 1.3, Entrada al Zoológico",
	"physical_city": "Mayaguez",
	"physical_state": "Puerto Rico",
	"physical_zip": "680",
	"update_url": "https://docs.google.com/forms/d/1slWOvxuLpuPdvDihSibLQq9BPsOqPzK8Hh93zCW3dRI/viewform?entry.823080433=the+collection+is+already+in+the+list&entry.764919322=urn:uuid:f2f50ab7-ac4a-430e-9c15-7e389a4bffdf&entry.326174790=University%20of%20Puerto%20Rico%20Mayaguez&entry.2031121141=University%20of%20Puerto%20Rico%20Mayaguez%20Invertebrate%20Collection&entry.4068754=UPRM&entry.1582913154=INVCOL&entry.1336841557=http://alexvandam.wix.com/uprminvcol&entry.103879345=https://www.idigbio.org/portal/recordsets/2b89de41-42bd-46c6-ab61-d386f855f7fb&entry.107456176=Mostly%20the%20Caribbean%20Islands%0A&entry.879476273=Arthropoda%20and%20Mollusca&entry.1321049572=Alex%20Van%20Dam&entry.1687847097=Director&entry.1086198428=alex.vandam@upr.edu&entry.246950189=Departamento%20de%20Biolog%C3%ADa,%20Call%20Box%209000&entry.1584255348=Mayaguez&entry.1966582743=Puerto%20Rico&entry.256217142=00681-9000&entry.447546773=Departamento%20de%20Biologia,%20Carr.%20108%20Barrio%20Miradero%20Km%201.3,%20Entrada%20al%20Zool%C3%B3gico&entry.1565624766=Mayaguez&entry.1920508789=Puerto%20Rico&entry.1022645685=680&entry.1499949381=",
	"lat": 0,
	"lon": 0
}
``` 
About this update: 
Alex Van Dam
alex.vandam@upr.edu

**Comment by kevinlove** [comment]
@JoannaMcCaffrey Did you ask this user to submit the form again? I just closed out issue #166 and the user should be able to validate changes in the beta-portal:

https://beta-portal.idigbio.org/collections/f2f50ab7-ac4a-430e-9c15-7e389a4bffdf

For now, I'm just going to mark this as a duplicate.

**Comment by JoannaMcCaffrey** [comment]
@kevinlove Nope - I told him I would take care of the changes in the backend. Gotta give it to him for his enthusiasm!
